test.js

```diff
@@ -1,57 +1,57 @@
+import {Writable as WritableStream} from 'stream';
 import test from 'ava';
-import hookStd from 'hook-std';
 import Ora from './';
 
 const spinnerChar = process.platform === 'win32' ? '-' : '⠋';
 
-function readOutput(spinner, callback) {
-	let out = '';
-
-	const unhook = hookStd.stderr({silent: true}, output => {
-		out += output;
+test.beforeEach(t => {
+	const noop = () => {};
+	const stream = new WritableStream({
+		write(chunk, encoding, callback) {
+			t.context.output += chunk.toString();
```

Comment:
@sindresorhus
Sorry, my forced push broke this conversation.
 
You said:
`.toString should be called at the end. Currently it can corrupt multi codepoint unicode sequences.`

I'm still confused about it.

[Here for example](https://github.com/jasonpincin/stream-to-string/blob/master/index.js#L14), is doing something very similar: